Sources/TokamakCore/Fiber/Layout/StackLayoutComputer.swift

```diff
@@ -0,0 +1,156 @@
+// Copyright 2022 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 5/24/22.
+//
+
+import Foundation
+
+final class StackLayoutComputer: LayoutComputer {
+  let proposedSize: CGSize
+  let axis: Axis
+  let alignment: Alignment
+  let spacing: CGFloat
+  /// Will contain a value of `1` for the axis that takes the size of the largest child.
+  /// For instance, a `VStack` will provide `1` for the width because
+  /// it takes the width of its largest child.
+  ///
+  /// Typically flipped from the width/height of `fitAxis`
+  let maxAxis: CGSize
```

Comment:
It's a bit unclear to me though why this type is `CGSize` if it contains a value of `1` according to the comment. Is that `1` encoded in the size somehow? Why so?

Reply:
That is, if possible values are only `1` or `0`, shouldn't it be something like `(width: Bool, height: Bool)`?

Reply:
Great, thanks!